ansible/add-ssh-keys.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Taken from: https://gist.github.com/EntropyWorks/a768b3bc4444146d56be81af05d73fed
+# Original idea found at http://stackoverflow.com/a/39083724
+#
+#   ansible -i inventory.ini add-ssh-keys.yml
+#
+- name: Store known hosts of 'all' the hosts in the inventory file
+  hosts: localhost
+  connection: local
+  vars:
+    ssh_known_hosts_command: "ssh-keyscan -T 10"
+    ssh_known_hosts_file: "../environments/{{ deploy_env }}/known_hosts"
+    ssh_known_hosts: "{{ groups['all'] }}"
+
+
+  tasks:
+  - name: For each host, scan for its ssh public key
+    shell: "{{ ssh_known_hosts_command }} -p {{ hostvars[item]['ansible_port']|default('22') }} {{ item }},`dig +short {{ item }}`"
+    with_items: "{{ ssh_known_hosts }}"
+    register: ssh_known_host_results
+    check_mode: no
+    ignore_errors: yes
+    tags:
+      - ssh
+
+  - name: Remove the public key in the '{{ ssh_known_hosts_file }}'
+    known_hosts:
+      name: "{{ item.item }}"
+      state: "absent"
+      path: "{{ ssh_known_hosts_file }}"
+    with_items: "{{ ssh_known_host_results.results }}"
+    tags:
+      - ssh
+
+  - name: Debug
+    debug: msg="b {{ item }}"
+    with_items: "{{ ssh_known_host_results.results }}"
+
+  - name: Add/update the public key in the '{{ ssh_known_hosts_file }}'
+    known_hosts:
+      name: "{{ item.item }}"
+      key: "{{ item.stdout }}"
+      state: "present"
+      path: "{{ ssh_known_hosts_file }}"
+    with_items: "{{ ssh_known_host_results.results }}"
+    tags:
+      - ssh
+
+  - name: For each host, ssh-copy-id my ssh public keys to the host
```

Comment:
I think we don't step, right? Since we handle ssh keys in a central place already? Maybe I'm misunderstanding what this step does.

Reply:
yeah, my bad, forgot to simply delete it from this gist